examples/Demo/Shared/Shared/DemoSearch.razor

```diff
@@ -1,24 +1,17 @@
-﻿<FluentSearch @bind-Value="@_searchValue"
-              Immediate
-              @bind-Value:after="HandleSearchInput"
-              Id="demo-search"
-              Placeholder="Search everything..."/>
-
-@if (_searchResults is not null)
-{
-    <FluentMenu Anchor="demo-search"
-                HorizontalPosition="HorizontalPosition.Right"
-                @bind-Open="_visible">
-        @foreach (var searchResult in _searchResults)
-        {
-            var item = NavProvider.FlattenedMenuItems.First(x => x.Title == searchResult);
-
-            <FluentMenuItem OnClick="@(() => HandleSearchClicked(item))">
-                <span slot="start">
-                    <FluentIcon Value="@(item.Icon)" Class="search-result-icon" Color="Color.Neutral" Slot="start"/>
-                </span>
-                @item.Title
-            </FluentMenuItem>
-        }
-    </FluentMenu>
-}
+﻿<FluentAutocomplete TOption="NavItem"
```

Comment:
You must "force" the width of the component to avoid the visual effect of "displacement".

![peek](https://github.com/microsoft/fluentui-blazor/assets/8350694/785050ef-eafe-448d-acbd-2bace8afe224)

Reply:
done